Add list and character component

buentellopc opened the pull request:
We will retrieve the data in order to fulfill the List and Character components.

Using the following API: https://rickandmortyapi.com/documentation/#rest

buentellopc committed:
Add list and character component

buentellopc committed:
Add another character

frxncismor commented on src/components/List.js:
```diff
@@ -0,0 +1,13 @@
+    <h2>Characters</h2>;
+    <Character />
+    <Character />
+    <Character />
```
We don't need this third character line, can you please delete it?
```suggestion
  
```

buentellopc committed:
Delete extra Character

Co-authored-by: Francisco J. Moreno <30633617+frxncismor@users.noreply.github.com>

buentellopc committed:
Add Character and List functionality to retrieve data from API

buentellopc committed:
Merge branch 'feature-add-list' of github.com:buentellopc/chapter1-spa into feature-add-list

buentellopc committed:
Add styles to List and Character components

frxncismor commented on src/components/Character.css:
```diff
@@ -0,0 +1,24 @@
+.card {
+  /* border: 1px solid red; */
```
Please delete the code you aren't using anymore in order to keep code clean
```suggestion
  
```

buentellopc committed:
Add Search component
-add styles for search component
-add functionality to List component to filter out characters

buentellopc commented on src/App.js:
```diff
@@ -1,23 +1,24 @@
 import "./App.css";
+import List from "./components/List";
+import Search from "./components/Search";
+import ParentSearch from "./components/ParentSearch";
```
```suggestion

```

buentellopc committed:
Update src/App.js

buentellopc committed:
Add scrolling functionality

buentellopc committed:
Merge branch 'feature-add-list' of github.com:buentellopc/chapter1-spa into feature-add-list

frxncismor committed:
Update src/components/Character.css

frxncismor commented on src/App.js:
```diff
@@ -1,23 +1,25 @@
+        <Search pattern={pattern} patternHandler={patternHandler} />
       </header>
+      <List pattern={pattern} />
+      {/* <InfiniteScroll /> */}
```
```suggestion

```

frxncismor committed:
Delete: infinite scroll component which is not being used
